apollo-coroutines-support/src/main/kotlin/com/apollographql/apollo/coroutines/CoroutinesExtensions.kt

```diff
@@ -19,7 +22,7 @@ import kotlinx.coroutines.flow.*
  * @return a flow which emits [Responses<T>]
  */
 @ExperimentalCoroutinesApi
-fun <T> ApolloCall<T>.toFlow() = callbackFlow {
+fun <T> ApolloCall<T>.toFlow(): Flow<Response<T>> = callbackFlow {
```

Comment:
Isn't type inference supposed to work here ? Or are you adding this for readability reasons ?

Reply:
I think originally this was because of `compileOnly`. I added it. But then didn't remove it since It is a good practice to explicitly define types in public APIs.